arch/arm/include/arch/config.h

```diff
@@ -6,7 +6,17 @@
 #ifndef XTF_ARM_CONFIG_H
 #define XTF_ARM_CONFIG_H
 
+/*
+ * On MMU less system, when using XTF as dom0 we need to know the load address
```

Comment:
I am not sure why you mention specifically dom0. This is also true for a domain because we don't guarantee that XTF will be loaded at the exact same place on each Xen versions.

Also, we may have spoke about it in the past but I can't remember the outcome, running XTF without MMU is going to require quite some works because Xen will work against you (we map the memory cached). So I am not sure we want to support MMU off use case.

Reply:
I agree that maintaining MMU off use case is hard. I'm ok to drop it as the code will be cleaner but I need others opinion.
At first I thought it'll be good to test Xen with MMU off domain because currently I do not see such scenario being tested.

Reply:
> At first I thought it'll be good to test Xen with MMU off domain because currently I do not see such scenario being tested.

They are not supported on Xen. Do you have any requirements to run workload with MMU off (other than the early boot)?

Reply:
We'd need MMU-less so that in the future we can change it to MPU domain.